README.md

```diff
@@ -62,22 +62,29 @@ Usage: docker volume create [OPTIONS]
 -d, --driver    string    Specify volume driver name (default "local")
 --label         list      Set metadata for a volume (default [])
 --name          string    Specify volume name
--o, --opt       map       Set driver specific options (default map[]) 
+-o, --opt       map       Set driver specific options (default map[])
```

Comment:
The original looks okay to me. In RAW format it's showing me 
https://raw.githubusercontent.com/projectatomic/docker-lvm-plugin/master/README.md

`-o, --opt       map       Set driver specific options (default map[])`

Not sure why you are seeing two `-`. The extra space can still be removed.

Reply:
Aah my bad ! You had just removed the extra space at the end of the line. Please ignore this comment.